docs/1___overview.adoc

```diff
@@ -124,6 +124,9 @@ In some ways the Scheduled Execution interface has similarities to the Model Exc
 [#figure-scheduled-execution-data-flow]
 image::images/scheduled-execution-data-flow.svg[width=40%, align="center"]
 
+_[An external scheduler is only required if the importer needs direct control over the tasks, e.g. to avoid overruns in a real-time application.
+To synchronize events between the importer and across FMUs, <<Clock, Clocks>> should be used that are available in all interface types.]_
```

Comment:
" The FMU provides model partitions defined by clocks in the FMU interface description. This way the importer can allow to configure the scheduling of each model partition, e.g. it enables to synchronize the execution across FMUs."